### PR TITLE
Peer20-253

### PR DESCRIPTION
**Pull request by alexeishulga**
Removed "Telos" from sign up pages

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   "app.common.showLess": "Show less {value}",
   "app.common.poweredByTelos": "Powered by {image}",
   "app.common.poweredByPeeranha": "Powered by {image}",
-  "app.common.termsOfTelos": "Telos Operating Agreement",
+  "app.common.termsOfTelos": "Operating Agreement",
```
Comment (ruslanzharkov) [inline app/translations/en.json:114]: `termsOfTelos` translation key should be changed to a new one

File: app/translations/ru.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
   "app.common.showLess": "Показать меньше {value}",
   "app.common.poweredByTelos": "Powered by {image}",
   "app.common.poweredByPeeranha": "Powered by {image}",
-  "app.common.termsOfTelos": "Telos Operating Agreement",
+  "app.common.termsOfTelos": "Operating Agreement",
```
Comment (ruslanzharkov) [inline app/translations/ru.json:114]: key should be another one, also russian translation here, not english

File: app/translations/en.json (hunk 1 above)
Comment (alexeishulga) [reply app/translations/en.json]: changed: https://github.com/peeranha/peeranha-web/pull/650/commits/fcf50b44b1470726b3dd53763fca278224a03a17

File: app/translations/ru.json (hunk 2 above)
Comment (alexeishulga) [reply app/translations/ru.json]: changed: https://github.com/peeranha/peeranha-web/pull/650/commits/fcf50b44b1470726b3dd53763fca278224a03a17

**Review by ruslanzharkov — APPROVED** [review]

**Pull request merged**